mne/annotations.py

```diff
@@ -1140,7 +1140,7 @@ def _write_annotations_txt(fname, annot):
         np.savetxt(fid, data, delimiter=",", fmt="%s")
 
 
-def read_annotations(fname, sfreq="auto", uint16_codec=None):
+def read_annotations(fname, sfreq="auto", uint16_codec=None, encoding_edf="utf8"):
```

Comment:
We don't say `uint16_codec_eeglab` so we might as well not say `encoding_edf`
```suggestion
def read_annotations(fname, sfreq="auto", uint16_codec=None, encoding="utf8"):
```
Also future compatible with possible need to pass `encoding` to any other readers... and there's no point in having a `encoding_edf=..., encoding_other=...` because only one will ever be used at a time internally.